roles/dosomething.phoenix/templates/nginx/app.j2

```diff
@@ -30,6 +30,7 @@ server {
   location / {
     # This is cool because no php is touched for static content
     try_files $uri @rewrite;
+    error_page 404 /leftathetrafficlight.php;
```

Comment:
Also, can you try if `error_page 404 = @rewrite;` will work as intended:
http://nginx.org/en/docs/http/ngx_http_core_module.html#error_page